### DISCUSSION TITLE
默认语言应该是中文，如果像网页一样，默认英文

### DISCUSSION THREAD

**Discussion by ghostvarg** [zhayujie/chatgpt-on-wechat#290] [Q&A]
默认语言应该是中文，如果像网页一样，默认英文，可以通过设置实现吗？感觉网页上的回复好像更丰富，更智能一些！

**Comment by zhayujie** [comment]
目前openai 接口的模式，和网页模型不同

不过可以在character_desc配置中通过英文设置来默认英文

**Reply by ghostvarg** [reply]
谢谢，我试下，请问怎么重启服务呢？

**Reply by zhayujie** [reply]
ps -ef | grep app.py | grep -v grep 命令可查看运行于后台的进程，如果想要重新启动程序可以先 kill 掉对应的进程。

如果你下载了最新的代码，还可以使用脚本快速关闭服务:

```bash
cd scripts
./shutdown.sh 
./start.sh
```

**Reply by ghostvarg** [reply]
我尝试重启了一下，重新二维码登录的，英文还是没有起效果，请问更新程序怎么操作？重新clone一下就可以吗？问题可能比较小白一些多谢了。

**Reply by ghostvarg** [reply]
对，我修改的是config.json里面的character_desc 内容修改成英文描述了，应该没有操作错误吧？

**Reply by zhayujie** [reply]
> 对，我修改的是config.json里面的character_desc 内容修改成英文描述了，应该没有操作错误吧？

你可以在 character_desc 中告诉它用英文回答， 然后你用英文提问题
